force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +253,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
+        List<integer>  numbList = new List<integer>(),
 
         // Add the integers 1, 2, and 3 to the list
         // add 1
         // add 2
         // add 3
 
         // Return the populated list
-        return null; // Replace null with the variable you used to store the result
+        numblist.add(1);
+        numblist.add(2);
+        numblist,add(3);
+        return List; // Replace null with the variable you used to store the result
```

Comment:
Good work. To be more concise and save a few lines, you can use the List constructor like this. 

```
 public static List<Integer> createAndPopulateList() {
      return new List<Integer>{1, 2, 3};
    }
    ```